### ISSUE TITLE
Tiny undefined request body problem

### ISSUE THREAD

**Issue by jimmywarting** [bitinn/node-fetch#208]
Found out about this while building a test for another lib

```js
req = new Request('.')
await req.text() // Fails in node-fetch can't read property `on` of undefined
console.assert(req.bodyUsed === false, 'expected request.bodyUsed to be equal to false')
```

**Comment by TimothyGu** [comment]
Fixed in v2 branch. I'll release an v2-alpha.1 hopefully later today.

**Comment by TimothyGu** [comment]
@bitinn, would you mind make me an owner on npm?

**Comment by jimmywarting** [comment]
Did this also get fixed for the Response - same there

res = new Response()
res.text()

**Comment by TimothyGu** [comment]
`Response` got fixed two months ago in 64e239b09258c9433ecd57b755967f6cccafa5e2

**Comment by bitinn** [comment]
I will get to it today, thx for your work :)

(From my phone)
